composer.json

```diff
@@ -39,5 +39,11 @@
 		"wiki": "https://github.com/sybrew/the-seo-framework/wiki"
 	},
 	"minimum-stability": "dev",
-	"prefer-stable": true
+	"prefer-stable": true,
+	"config": {
+		"allow-plugins": {
+			"composer/installers": true,
+			"dealerdirect/phpcodesniffer-composer-installer": true
+		}
+	}
```

Comment:
Upgrade to Composer v2.
It is almost 3 years old.
https://github.com/composer/composer/releases/tag/2.0.0

Reply:
I don't use Composer for WordPress, only standalone stuff (like WPCS) -- this file is mostly set up by those who do use it inside WordPress, e.g., by using [Bedrock](https://roots.io/bedrock/), and I rely on their feedback when an upgrade is required 😅. 

I don't know what the config/allow-plugins would limit, but what you added seems standard.

Reply:
You have met your man.
Please see https://github.com/szepeviktor/composer-managed-wordpress